apps/native/app/src/ui/lib/barcode/barcode.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useMemo, useRef } from 'react'
+import { Animated, Easing } from 'react-native'
+import styled from 'styled-components/native'
+import Svg, { G, Path } from 'react-native-svg'
+import { createPDF417 } from '../../../lib/pdf417/pdf417-min'
```

Comment:
Is there a reason we can't import this from npm? I know Hermes has `crypto` issues

Maybe we can rather use something like [react-native-barcode-creator](https://github.com/VittoriDavide/react-native-barcode-creator) for native implementation?

Reply:
That was what I tried at first and until yesterday but Android was not behaving 😿

Reply:
Let me check if I can find a npm source, but the package I took it from is only on github
https://github.com/bkuzmic/pdf417-js